### PR TITLE
Test task.

### PR DESCRIPTION
**Pull request by molodora**
Add law of photoelectric effect and test for it.

### CODE REVIEW

File: test/waves/photoelectric_effect.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+from collections import namedtuple
```
Comment (blackyblack) [inline test/waves/photoelectric_effect.py:1]: Test's name should end with `test`

File: symplyphysics/laws/waves/photoelectric_effect.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+from sympy import (Eq, solve)
```
Comment (blackyblack) [inline symplyphysics/laws/waves/photoelectric_effect.py:1]: We have a convention of not using marketing names, like "Newton's first law" or such. Write law name as "what_we_get_from_what_we_put" or "value_is_something" for definitions.

**Review by blackyblack — APPROVED** [review]

File: symplyphysics/laws/waves/photoelectric_effect.py (hunk 2 above)
Comment (blackyblack) [reply symplyphysics/laws/waves/photoelectric_effect.py:1]: By the way, maybe this law should be in a `quantum` branch of laws?

File: test/waves/photoelectric_effect.py (hunk 1 above)
Comment (molodora) [reply test/waves/photoelectric_effect.py]: Changed filename to `photoelectron_energy_from_frequency_test.py`

**Review by blackyblack — APPROVED** [review]

File: symplyphysics/laws/waves/photoelectric_effect.py (hunk 2 above)
Comment (molodora) [reply symplyphysics/laws/waves/photoelectric_effect.py]: Changed filename to `photoelectron_energy_from_frequency.py`.

It seems to me, there're several files that should be in `quantum` folder, such as `photon_energy...` or `photon_momentum...`. Or may be it shoud be named `atomic`, for both particles and waves?

**Pull request merged**